README.md

```diff
@@ -269,6 +271,36 @@ let () =
 *)
 ```
 
+or use `Js.String`
+
+```ml
+let code = {|
+  let borderLeftColor = "borderLeftColor";
+  let borderRightColor = "borderRightColor";
+|}
+
+let matchLetter letter _ _ = "-" ^ (Js.String.toLowerCase letter)
+
+let matchLiterals _ p1 _ _ =
+  let t =
+    Js.String.unsafeReplaceBy0
+      [%re "/[A-Z]/g"] matchLetter p1 in
+  {j|"$t"|j}
+
+let () =
+  (code |>
+     Js.String.unsafeReplaceBy1
+        [%re
+           "/\"([^\"]*)\"/g"]
```

Comment:
This formatting is a bit weird. I don't think the line break is particularly necessary here.